examples/text-generation/README.md

```diff
@@ -238,6 +240,15 @@ Another way to simulate dynamic input is to use `--simulate_dyn_prompt`. For exa
 
 While `--bucket_size` works for any model without model file changes, an even more optimized version of bucketing is supported for certain models like Llama. This can be enabled by setting `--bucket_internal` flag (along with `--bucket_size` to specify the bucket size)
 
+
+### Running with torch.compile
+
+torch.compile is an experimental feature. It has not been validated for all models. To enable torch.compile, please
+set the following environment variables before running the command: `PT_ENABLE_INT64_SUPPORT=1` and `PT_HPU_LAZY_MODE=0`.
+
+You will also need to add `--torch_compile` in your command.
```

Comment:
you also need to add  "--torch_compile_backend hpu_backend" for runnin compile on HPU.

Reply:
Hmm this is in the training args, but the text-generation example doesn't rely on them. It just has a `torch_compile` arg that calls `torch.compile` with the right backend here: https://github.com/huggingface/optimum-habana/blob/9f4c808a720a32747081a2d0c6124f5bad96d94b/examples/text-generation/utils.py#L154
Does that look right?